cloud.foundry.cli/src/test/java/cloud/foundry/cli/crosscutting/mapping/YamlMapperTest.java

```diff
@@ -1,48 +1,114 @@
 package cloud.foundry.cli.crosscutting.mapping;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.*;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import cloud.foundry.cli.crosscutting.exceptions.RefResolvingException;
 import cloud.foundry.cli.crosscutting.mapping.beans.ApplicationBean;
 import cloud.foundry.cli.crosscutting.mapping.beans.ServiceBean;
 import cloud.foundry.cli.crosscutting.mapping.beans.SpecBean;
 import org.junit.jupiter.api.Test;
 import org.yaml.snakeyaml.constructor.ConstructorException;
+import org.yaml.snakeyaml.parser.ParserException;
+import org.yaml.snakeyaml.scanner.ScannerException;
 
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.Map;
 import java.util.Map.Entry;
-import java.util.HashMap;
 
 
 public class YamlMapperTest {
 
+    private static final String SERVICE_YAML_EXAMPLE =
+                    "service: someService\n" +
+                    "tags:\n" +
+                    "- tag1\n" +
+                    "- tag2\n";
+
     @Test
-    public void dumpTest() {
-        //given
-        Map<String, ServiceBean> services = new HashMap<>();
+    public void testDump() {
+        // given
         ServiceBean serviceBean = new ServiceBean();
+
         serviceBean.setService("someService");
         serviceBean.setPlan(null);
         serviceBean.setTags(Arrays.asList("tag1", "tag2"));
-        services.put("serviceName", serviceBean);
+
+        // when
+        String dump = YamlMapper.dump(serviceBean);
+
+        // then
+        assertThat(dump, is(SERVICE_YAML_EXAMPLE));
```

Comment:
Why create a static variable for the string here but not in the other tests? I think you could just move it into this method, too.

Reply:
This string is used in various test cases, that is why I moved it into a static variable (to reduce duplicate code). We could also come up with different examples per test case if that is more desirable.